aspnet/publishing/iis.rst

```diff
@@ -4,23 +4,37 @@ Publishing to IIS
 By `Rick Anderson`_
 
 In this article:
+	- `Configure the IIS server`_
 	- `Publish from Visual Studio`_
-	- `Xcopy to IIS Server`_
+	- `Xcopy to IIS server`_
+	- `Supported operating systems`_
+	- `Common errors`_
 	- `Additional Resources`_
-	
-Publish from Visual Studio  
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-1. Create an ASP.NET 5 app. In this sample, I'll create a MVC 6 app using the **Web Site** template under **ASP.NET 5 Preview Templates**. If you're not using the ``web`` and ``gen`` commands in your development and production work flow, you can remove them from the *project.json* file.
 
-.. code-block:: javascript
+Configure the IIS server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Install the HTTP Platform Handler:
+
+	- `64 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkID=690721>`_
+	- `32 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkId=690722>`_
+
+- Enable the **Web Server (IIS)** role and **Application Development** role services. Skip this step if you are on Windows 7 through Windows 10 (Client OSes), it's required only for servers.
```

Comment:
@Tratcher Can you confirm that the only thing Application Development will do for `dnx451` and `dnxcore50` apps is make it so that IIS can read the `<handlers>` section of web.config? If so, I'm preferring (just for my own stuff) to not install the .NET 4.5 and ISAPI bits, since they won't affect apps that aren't using any part of .NET 4.x, and I can unlock that section of `web.config` with the `Configuration Editor` at the server level in IIS. I'm not suggesting the docs should suggest one approach or the other (or both), I'm just interested in what specific cases would require the Application Development role under .NET 5 ... would it be `net451` and `net46` only need that, while `dnx46` and `dnxcore50` don't need it?